src/main/java/com/courtcanva/ccfranchise/dtos/StaffInfoDto.java

```diff
@@ -0,0 +1,52 @@
+package com.courtcanva.ccfranchise.dtos;
+
+import com.courtcanva.ccfranchise.constants.State;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class StaffInfoDto {
+
+    @NotNull
+    private String firstName;
+
+    @NotNull
+    private String lastName;
+
+    @NotNull
+    private String franchiseAbn;
```

Comment:
I don't think this is part of `Staff` table?

Reply:
I made a mistake, thx.